test/framework/e2e_framework.go

```diff
@@ -598,6 +598,37 @@ func (f *E2eFramework) DumpClusterInfo(test string, namespace ...string) error {
 		if err := kubectl.DumpClusterInfo(opts); err != nil {
 			errs = append(errs, fmt.Errorf("failed to dump cluster info for cluster %s: %w", ctx, err))
 		}
+
+		// Store the list of pods in an easy to read format.
+		output, err := kubectl.Get(kubectl.Options{Context: ctx, Namespace: namespace[1]}, "pods", "-o", "wide")
```

Comment:
For tests with cluster-scoped deployments, we have objects spread across multiple namespaces.

Reply:
I've changed this to loop over all namespaces passed to `DumpClusterInfo`.
Let me know if that works for you.